x/marker/client/cli/query.go

```diff
@@ -244,10 +244,12 @@ func MarkerEscrowCmd() *cobra.Command {
 }
 
 // MarkerSupplyCmd is the CLI command for querying marker module registrations.
+// Deprecated: This command returns initial/target supply, not actual circulating supply.
```

Comment:
No need to deprecate this `MarkerSupplyCmd` function since it's not a public-facing thing.